test/e2e/data/shared/v1.10/metadata.yaml

```diff
@@ -1,6 +1,12 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+- major: 1
+  minor: 10
+  contract: v1beta1
+- major: 1
```

Comment:
Why missing this was not an issue before?

Reply:
shared/main/metadata.yaml is not used anywhere, so it does not matter what we had in there before. That's why I have deleted it.